### PR TITLE
Project-weather-app by Joel and Sofia

### PR DESCRIPTION
**Pull request by joeohm**
https://helpful-pony-3b978e.netlify.app/

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+        sectionForecast.innerHTML += `
+      <div class="forecast-day-container">
+       <p class="forecast-day">${weekday}</p>
+       <p class="forecast-day-temp">${day.main.temp.toFixed(1)}°</p>
+      </div>
+      `;
+      });
+    });
+}
```
Comment (lindanorberg) [inline script.js:108]: Nicely done! Short and tidy!

File: style.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  font-size: 35px;
+  font-weight: bold;
+  padding-bottom: 30px;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  width: auto;
+  height: 100%;
```
Comment (lindanorberg) [inline style.css:41]: The main cointaner is not covering the full screen. 

Maybe try to add: min-height: 100%;

File: style.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+#164A68 blue text rainy
+#A3DEF7 blue background rainy
+#2A5510 green text sunny
+#F7E9B9 yellow background sunny
+*/
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+
+body {
+  font-family: 'Montserrat', sans-serif;
```
Comment (lindanorberg) [inline style.css:21]: Try to add: height: 100%; 
And maybe target html as well.

File: style.css (hunk 2 above)
Comment (lindanorberg) [reply style.css:41]: https://stackoverflow.com/questions/6654958/make-body-have-100-of-the-browser-height

File: style.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  flex-direction: column;
+  width: auto;
+  height: 100%;
+  padding-bottom: 20px;
+}
+
+/*** City picker section ***/
+
+select {
+  color: unset;
```
Comment (lindanorberg) [inline style.css:48]: Thank you for learning us this! We understand that this makes the select forget the previous style, but we don't understand where the select picks up the new style. . .

File: style.css [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+.city-select-clear {
+  background-color: #fff5d5;
+}
+.city-select-clouds {
+  background-color: #ffffff;
+}
+.city-select-rain {
+  background-color: #c3ebfc;
+}
```
Comment (lindanorberg) [inline style.css:78]: Nice feature to choose a lighter colour of the select background.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+.rain {
+  background-color: #a3def7;
+  color: #164a68;
+}
+
+/*** For Tablet ***/
+
+@media (min-width: 768px) and (max-width: 991px) {
```
Comment (lindanorberg) [inline style.css:127]: You have the same styling for h1 in both tablet och desktop. If max-width is removed from tablet, the styling goes for desktop as well (exept for the new styling you ad under desktop). And then you dont have to write styling for h1 twice.

**Review by lindanorberg — COMMENTED** [review]
Nice work! Tidy code! 

We understand the code, the code does not have any duplicates, it works and follows the general requirements. Except for not following the design instructions regarding background color which should stretch al over the browser window.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
[the remark this answers is not in the document]
Comment (lindanorberg) [reply index.html:6]: Oops, now I see that this is part of starting html code. Just ignore this comment. . .

File: style.css (hunk 2 above)
Comment (joeohm) [reply style.css:41]: Thanks for the tip and link!
min-height was not needed on main-container in the end when the height and min-height of body/html was set to 100%.

File: style.css (hunk 3 above)
Comment (joeohm) [reply style.css:21]: Thanks for the tip, it worked!
```
body,
html {
  font-family: 'Montserrat', sans-serif;
  height: 100%;
  min-height: 100%;
}

File: style.css (hunk 2 above)
Comment (joeohm) [reply style.css:43]: Actually min-height was not needed on body/html either, only height: 100%

**Pull request closed without merging**